Sources/Sentry/SentrySystemEventsBreadcrumbs.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+#if TARGET_OS_IOS
+#    import <UIKit/UIKit.h>
+#endif
+
+@interface SentrySystemEventsBreadcrumbs : NSObject
+
+- (void)start;
+
+#if TARGET_OS_IOS
+- (void)start:(UIDevice *)currentDevice;
```

Comment:
only for tests, is there such @TestOnly annotation?

Reply:
`m`: We don't have annotations in Objective-C. I'm not a big fan of @testonly annotations anyway. Why not just have `- (void)start:(UIDevice *)currentDevice;` and remove the `- (void)start;`?

Reply:
because we don't need to force people to pass a parameter that we are able to get anyway.
it's only exposed for testing.

Reply:
But now I'm confused because there are two different methods. Which one should I use? What is the difference? I would prefer having only one method and going for one of the solutions I mentioned above. I don't have a strong opinion on this, but if we keep it as it is at least we should add a comment stating that this is only there for testing or something similar.